spk/octoprint/src/config.yaml

```diff
@@ -3,3 +3,13 @@ serial:
 
 server:
   port: 8088
+
+plugins:
+  cura:
+    cura_engine: /usr/local/curaengine/bin/CuraEngine
```

Comment:
Not unless somebody tries to slice an STL file.

There's also a big "test" button for the CuraEngine binary path in the upcoming 1.3.0 release, so less chance for user error there.